src/test/java/com/epam/test_generator/dao/SuitDAOTest.java

```diff
@@ -136,9 +136,75 @@ public void testUpdateName() {
         Assert.assertEquals(newSuit, suitDAO.getOne(id));
     }
 
+    @Test
+    public void testUpdateWithRowNumbers() {
+
+        final List<Suit> expectedSuits = suitDAO.save(getExpectedSuitsFromDatabase());
```

Comment:
Expected value is constant in unit tests. Not something obtained from database, but something hardcoded in test method body.